packages/now-build-utils/src/fs/node-version.ts

```diff
@@ -3,11 +3,12 @@ import { NodeVersion } from '../types';
 
 const supportedOptions: NodeVersion[] = [
   { major: 10, range: '10.x', runtime: 'nodejs10.x' },
-  { major: 8, range: '8.x', runtime: 'nodejs8.10' },
+  { major: 8, range: '8.10.x', runtime: 'nodejs8.10' },
 ];
 
 // This version should match Fargate's default in the PATH
-export const defaultSelection = supportedOptions[supportedOptions.length - 1];
+// Today that is Node 8
+export const defaultSelection = supportedOptions.filter(o => o.major === 8)[0];
```

Comment:
```suggestion
export const defaultSelection = supportedOptions.find(o => o.major === 8);
```

Will automatically pick the first hit.